core/worker.go

```diff
@@ -512,7 +518,9 @@ func (w *worker) GeneratePendingHeader(block *types.WorkObject, fill bool) (*typ
 		w.coinbase = common.Zero
 	}
 	coinbase = w.coinbase // Use the preset address as the fee recipient
-
+	if _, err := coinbase.InternalAddress(); nodeCtx == common.ZONE_CTX && err != nil {
+		return nil, fmt.Errorf("invalid coinbase address %v: %v", coinbase, err)
```

Comment:
Can you please make an error variable for this so it doesn't get repeated individually.

Reply:
An error variable is a constant, so it would not include the underlying err